src/routes/transactions/__tests__/controllers/list-incoming-transfers-by-safe.transactions.controller.spec.ts

```diff
@@ -171,50 +191,80 @@ describe('List incoming transfers by Safe - Transactions Controller (Unit)', ()
       .get(`/v1/chains/${chainId}/safes/${safeAddress}/incoming-transfers/`)
       .expect(200)
       .then(({ body }) => {
-        expect(body).toEqual(
-          getJsonResource('incoming-transfers/erc20/expected-response.json'),
-        );
+        expect(body).toMatchObject({
+          results: [
+            {
+              type: 'TRANSACTION',
+              transaction: {
+                id: 'transfer_0x4127839cdf4F73d9fC9a2C2861d8d1799e9DF40C_e1015fc6905',
+                executionInfo: null,
+                safeAppInfo: null,
+                timestamp: erc20Transfer.executionDate.getTime(),
+                txStatus: 'SUCCESS',
+                txInfo: {
+                  type: 'Transfer',
+                  sender: { value: safe.address },
+                  recipient: { value: safe.address },
+                  direction: 'OUTGOING',
+                  transferInfo: {
+                    type: 'ERC20',
+                    tokenAddress: erc20Transfer.tokenAddress,
+                    tokenName: token.name,
+                    tokenSymbol: token.symbol,
+                    logoUri: token.logoUri,
+                    decimals: token.decimals,
+                    value: erc20Transfer.value,
+                  },
+                },
+              },
+              conflictType: 'None',
+            },
+          ],
+        });
       });
   });
 
   it('Should get a ERC721 incoming transfer mapped to the expected format', async () => {
     const chainId = faker.string.numeric();
     const safeAddress = faker.finance.ethereumAddress();
     const chain = chainBuilder().with('chainId', chainId).build();
+    const safe = safeBuilder()
+      .with('address', '0x4127839cdf4F73d9fC9a2C2861d8d1799e9DF40C')
```

Comment:
```suggestion
    const safeAddress = '0x4127839cdf4F73d9fC9a2C2861d8d1799e9DF40C';
    const chain = chainBuilder().with('chainId', chainId).build();
    const safe = safeBuilder()
      .with('address', safeAddress)
```

Reply:
As commented in https://github.com/safe-global/safe-client-gateway-nest/pull/503#discussion_r1243337206, I'll take the chance to remove this concrete ethereum address reference for the sake of readability, but thank you for your suggestions! 😄

Reply:
Shall we also pass the generated address to the `safeBuilder` though?

Reply:
Changed in [0450df9](https://github.com/safe-global/safe-client-gateway-nest/pull/503/commits/0450df9f6fa71b7b2f5bd8b364e0fa0aedbdb2f3)